cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateMaskedPaths(spec *rspec.Spec) error {
+	fmt.Println("validating maskedPaths")
+	for _, v := range spec.Linux.MaskedPaths {
+		fi, err := os.Stat(v)
+		if err != nil {
+			return err
+		}
+		if fi.Mode()&0444 != 0 {
+			return fmt.Errorf("%v should be readonly", v)
```

Comment:
On Fri, May 27, 2016 at 12:30:03AM -0700, Ma Shimiao wrote:
> spec says "mask over the provided paths inside the container so that
> they cannot be read."  it definitely says change paths permission to
> make them can't be read.

I don't read “mask over” as clearly “change the permission bits”.  I
haven't been able to turn up runC's implementation with grep, but
@mrunalp's original runtime-spec PR was clearly recommending
bind-mounts [1].

[1]: https://github.com/opencontainers/runtime-spec/pull/186/files